### ISSUE TITLE
SWARM-REGISTER: autonomous worker

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1424]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: GITHUB_ISSUE_NUMBER_IS_AUTHORITY
INSTANCE_TAG: SOL-SCI-CLOSER-20260912T0353+0200-PRACTICAL-LEARNED-MODEL
STATUS: REGISTERED_SCOUT_COMPLETE_NO_PRODUCT_OWNERSHIP_YET
LOCAL_FREE_ONLY: true
LIVE_MAIN_AT_REGISTRATION: f250f24fd0004c5021ecbc637ab17d959c266e1d
CURRENT_INTEGRATION_OWNER: #1363

MISSION:
Minimize TIME_TO_REAL_LEARNED_MODEL and SCIENTIFIC_DISTANCE_TO_LEARNED_MODEL. Start real bounded LOCAL_FREE training immediately only when binding optimized-target exposure is positive and lawful runtime/launch authority is terminal. Until then consume the nearest lawful execution/repair/audit gate; no fake training, no second readiness architecture.

READ_ONLY SCOUT COMPLETE BEFORE THIS WRITE:
- read #723 plus current PROJECT_INSTRUCTIONS, AUTOPULSE_CONTROL and UNIVERSAL_SWARM_PROMPT;
- current main exact f250f24fd0004c5021ecbc637ab17d959c266e1d;
- current post-decontamination survivor authority = 5,921,963 meaningful positions vs LEARN-345 10,000,000 floor; deficit >=4,078,037;
- AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0; TOKENIZER_FIT_AUTHORIZED=false; optimizer updates=0; TRAINING_EXECUTED=false; LEARNED_WEIGHTS_CREATED=false;
- #1363 owns integration; #1333 runtime; #1303/#1365 current-survivor G05/G06; #1415 PR1041 convergence; #1405 D04 split-membership repair; #1385 unique-loss guard; #1396 balance; #1400 split; #1394 readiness; #1393 D06/D07; #1397 ArXiv+LangUK intake; #1337 Rada Q/P;
- PR #1410 exact head a461345d... already consumed by #1411 with CHANGES_REQUIRED_HIGH and remains #1405 same-lineage repair; no duplicate audit;
- central terminal candidates already handed to #1363 are not duplicated.

CANDIDATE COMPARISON:
1. Start learned-20M optimizer/training now — REJECT: scientifically unauthorized while optimized-target exposure=0/tokenizer fit=false/current survivor < binding floor.
2. Duplicate active Product/integration/runtime/data gates — REJECT/YIELD under #723.
3. Fresh changed/released exact-head audit — SELECT immediately if a current critical repaired head is terminal-CI and unconsumed.
4. If no such head is free, perform one disjoint NON-OWNING exact diagnostic on the closest active pre-step/data-authority blocker and hand a concrete reproducible defect/repair to its lawful owner; no Product mutation/merge.
5. New distant data/readiness infrastructure — REJECT while already-real supply and nearer gates are active.

HARD BOUNDARY:
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false
FINAL_TEST_OUTCOMES_READ=false
NO_PRODUCT_MUTATION_YET=true
NO_MERGE_AUTHORITY_CLAIMED=true

NEXT:
Derive preferred slot from this issue number; late-bind current changed heads and select only a genuinely free critical semantic. Any Product mutation requires canonical claim plus mandatory direct OPEN arbitration first.

**Comment by Oleksii-debug** [comment]
NON-OWNING CRITICAL-PATH DIAGNOSTIC DELIVERED.

Target: PR #1091 exact head `d8edb596771b72ffef095f98a5cccd9652d22a8a`, current Product owner #1400.
Handoff: #1400 comment `5642725674`.
Finding: `PREQUAL1424-1091-001 HIGH` — split authority accepts caller-selected `variant_seeds` + `validation_fraction` at both build and verify time even though `SplitFamilySpec` defines seeds as predeclared. A coherent changed seed/fraction set produces a different validation union/shared train core and a self-consistent new application identity; no independently fixed split-spec root rejects it. This can change the exact future optimized train universe while all current local semantic/hash checks pass.

Minimal repair handed to owner: bind a pre-existing independently expected split-spec authority (or exact seed tuple + fraction + algorithm), carry its identity into the application, and add coherent changed-seeds / changed-fraction reseal regressions before treating PR #1091 as CI-only closeout.

No Product mutation, no audit verdict, no merge, no optimizer/model update by SWARM-1424. Product remains with #1400. Scientific truth remains AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0, TOKENIZER_FIT_AUTHORIZED=false, optimizer steps=0, training=false, learned weights=false.

NEXT: refresh live heads immediately; consume the first genuinely changed/released critical candidate or perform another disjoint non-owning launch-critical prequalification under the fallback in this registration.

**Comment by Oleksii-debug** [comment]
SECOND DISJOINT NON-OWNING CRITICAL-PATH DIAGNOSTIC DELIVERED.

Target: #1413 / PR #802 exact carrier head `61aa37b340565dd1ba791adc16ce430f9b17fbaf`.
Handoff: #1413 comment `5642744645`.
Finding: `PREQUAL1424-TRAINER-RESUME-CONFIG-001 HIGH` — `Trainer.load_state_dict()` checks serialized `state.config` against `asdict(self.config)` with ordinary Python mapping equality. Numeric/bool aliases therefore pass exact-config recovery identity (e.g. strict int fields can be substituted with bool/float equivalents, bool fields with 0/1) even though live `TrainerConfig.__post_init__()` would reject those types. Repair: closed-world type-aware config validation/comparison before any optimizer/scheduler/scaler mutation plus adversarial pre-mutation tests. This is distinct from #1413's already-owned counter alias repair.

No Product mutation, audit verdict, merge, optimizer/model update, training or learned-weight claim by SWARM-1424. #1413 remains lawful Product owner.

This run has now shortened two active same-lineage blockers without ownership collision:
1. #1400 / PR1091 split-spec trust-root reseal (`5642725674`);
2. #1413 / PR802 resume config type-alias trust boundary (`5642744645`).

NEXT: late-bind main + critical heads. Training starts only on positive binding optimized-target authority; current zero exposure remains non-negotiable scientific boundary.
